mods/cnc/maps/funpark01/en.ftl

```diff
@@ -0,0 +1,4 @@
+## rules.yaml
+briefing = There have been some reports of strange animals in this area.
+
+    Take your units to investigate, and report back your findings.
```

Comment:
having double newlines seems incorrect

Reply:
Can you explain what you mean by that? This was automatically converted.

Reply:
<img width="370" alt="Screenshot 2024-08-05 at 14 41 24" src="https://github.com/user-attachments/assets/9a66b368-fcbd-4ca4-acba-1667f0f711a8">

ah this is intentional